adminpages/member-edit/pmpro-class-member-edit-panel-user-fields.php

```diff
@@ -7,20 +7,27 @@ class PMPro_Member_Edit_Panel_User_Fields extends PMPro_Member_Edit_Panel {
 	public function __construct( $field_group_name ) {
 		$this->slug = 'user_fields_' . sanitize_title( $field_group_name );
 		$this->title = $field_group_name;
-		$this->submit_text = __( 'Update Member', 'paid-memberships-pro' );
+		$this->submit_text = current_user_can( 'edit_users' ) ? __( 'Update Member', 'paid-memberships-pro' ) : '';
 	}
 
 	/**
 	 * Display the panel contents.
 	 */
 	protected function display_panel_contents() {
+		// Print the group description.
+		$field_group = pmpro_get_field_group_by_name( $this->title );
+		if ( ! empty( $field_group->description ) ) {
+			echo wp_kses_post( wpautop( $field_group->description ) );
```

Comment:
In core currently, we just use wp_kses_posts and don't wpautop. Do you think we should keep this the same as https://github.com/strangerstudios/paid-memberships-pro/blob/v3.0/includes/fields.php#L642 OR should we adjust that? I'm inclined to make sure they are the same.